elixir/apps/domain/lib/domain/relays.ex

```diff
@@ -270,10 +270,9 @@ defmodule Domain.Relays do
     {:ok, relays}
   end
 
-  def generate_username_and_password(%Relay{stamp_secret: stamp_secret}, %DateTime{} = expires_at)
+  def generate_username_and_password(%Relay{stamp_secret: stamp_secret}, salt, expires_at)
       when is_binary(stamp_secret) do
     expires_at = DateTime.to_unix(expires_at, :second)
-    salt = Domain.Crypto.random_token()
```

Comment:
cc @bmanifold - `salt` here is now the `url_base64(sha256(token_string))` so that portal connection flaps do not disrupt relayed connections.